hooks/post-command

```diff
@@ -45,19 +63,20 @@ main() {
   fi
 
   local ci_env
-  ci_env=$(bash <(curl -s -S --connect-timeout 10 --retry 3 --retry-delay 10 https://codecov.io/env))
+  ci_env=$(bash "${codecov_bash_dir}/env")
```

Comment:
Now that we don't pipe curl output anymore, would it make more sense to `chmod +x "${codecov_bash_dir}/env"` after download and then we can
```suggestion
  ci_env=$("${codecov_bash_dir}/env")
```
?